db/helpers/common.js

```diff
@@ -137,6 +139,17 @@ function createDBLog(inst, eventType, changedKeys, ignoreAttributes) {
 }
 
 /**
+ * A function to see if an object is a room object or not. It returns true
+ * if an object passed has 'type' and 'settings' as its property.
+ * @param  {Object}  obj - An object instance
+ * @returns {Boolean} - returns true if the object has the property
+ * "type" && "settings"
```

Comment:
This doesn't seem like a particularly strong type check. I *think* when you actually get a sequelize model instance, you should have a reference to the model itself. Can you check that?